hayeon/programmers-1/폰켓몬.js

```diff
@@ -0,0 +1,15 @@
+function solution(nums) {
+  const onlyPhoneketmon = [...new Set(nums)];
+
+  const choice = nums.length / 2;
+
+  return onlyPhoneketmon.length < choice ? onlyPhoneketmon.length : choice;
+}
```

Comment:
```suggestion
function solution(nums) {
  const setPhoneketmon = new Set(nums);
  const choice = Math.floor(nums.length / 2);

  return setPhoneketmon.size < choice ? setPhoneketmon.size : choice;
}
```
- set 객체를 통해서 중복 제거 후 결과를 도출한 풀이 잘봣습니다 :)
- set 인스턴스는 size라는 메서드가 존재합니다 해당 메서드를 이용해서 길이를 확인할 수 있습니다.
- 또한 자바스크립트는 나누기 연산을 하면 소수점까지 나와버리기 때문에 Math.floor을 이용해서 소수점을 제거하는 습관을 들이시는게 좋습니다. 🙇‍♂️

Reply:
넵 !! set객체 공부 제대로 해놓겠습니다! 감사합니다!!😆